### DISCUSSION TITLE
Use Sparkline chart in table

### DISCUSSION THREAD

**Discussion by vlowe85** [leandrocfe/filament-apex-charts#78] [Ideas]
Hi Leandro,

I haven't attempted this myself yet but thought i would ask first.

I see apex charts has a `Sparkline` option on its charts [here](https://apexcharts.com/javascript-chart-demos/sparklines/basic/)

I was wondering, based on how the plugin in integrated, how i could get a small sparkline chart into a table column. Presumably using a custom column?

Any thoughts or advice would be a good start. Thanks

**Comment by leandrocfe** [comment]
Custom Column. You can also use this https://filamentphp.com/plugins/lara-zeus-inline-chart
